xlive/H2MOD.cpp

```diff
@@ -229,7 +231,7 @@ void H2MOD::logToDedicatedServerConsole(wchar_t* message) {
 }
 
 int __cdecl dediCommandHook(int a1, int a2, int a3) {
-	h2mod->logToDedicatedServerConsole(L"Dedicated command\n");
+	//h2mod->logToDedicatedServerConsole(L"Dedicated command\n");
```

Comment:
why?

Reply:
Would be nice to log that only if a custom command has been used.